### PR TITLE
Document execute() methode for function test

### PR DESCRIPTION
**Pull request by baurmatt**
Fixes GH-719.

### CODE REVIEW

**Comment by baurmatt** [comment]
Failing tests seems unrelated to my change.

**Comment by rodjek** [comment]
Thanks @baurmatt !

File: README.md [hunk 1]
```diff
@@ -869,11 +869,11 @@ either the `with_params` chain method in the `run` matcher
 it { is_expected.to run.with_params('foo', 'bar', ['baz']) }
 ```
 
-Or by using the `call` method on the subject directly
+Or by using the `execute` method on the subject directly
 
 ```ruby
 it 'something' do
-  subject.call(['foo', 'bar', ['baz']])
+  subject.exectue('foo', 'bar', ['baz'])
```
Comment (alexjfisher) [inline README.md]: spelling

Comment (alexjfisher) [reply README.md]: Fixed in https://github.com/rodjek/rspec-puppet/pull/733

Comment (baurmatt) [reply README.md]: Sorry :(

**Pull request merged**